src/redux/books/books.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+const Books = () => (
+  <div className="d-flex flex-column justify-content-center mx-4">
+    <div className="d-flex flex-row justify-content-center">
+      <div className="d-flex flex-column px-3">
+        <h3>Lorem</h3>
+        <h1>Lorem Ipsun</h1>
+        <span>Subtittle</span>
+        <div>
+          <ul>
+            <li>Comments</li>
+            <li>Remove</li>
+            <li>Edit</li>
+          </ul>
+        </div>
+      </div>
+      <div className="px-3">
+        <div>Animated Circle</div>
+        <div>XX%</div>
+        <h4>Completed</h4>
+      </div>
+      <div className="px-3">
+        <h3>CURRENT CHAPTER</h3>
+        <p>Chapter 17</p>
+        <button type="button" className="btn btn-primary">Update Progress</button>
+      </div>
+    </div>
```

Comment:
- [ ] You should have a BookList component that renders a list of books using the Book component, so this means you need to create both the BookList component and the Book component. Separate this file to enable these changes. Remember that the Book component should take the book data as a prop and render the title and the author.
- [ ] Add these components to the components directory and not the redux directory